docs/source/examples/shapes_images_positions.rst

```diff
@@ -33,6 +33,39 @@ Shapes, Images and Positions
             self.play(Rotating(dot, about_point=[2, 0, 0]), run_time=1.5)
             self.wait()
 
+Download the resource for the following example `here <https://github.com/ManimCommunity/manim/blob/master/docs/source/_static/manim-logo-sidebar.svg>`_ 
+
+.. manim:: ManimCELogoFromSVG
+
+    class ManimCELogoFromSVG(Scene):
+        def construct(self):
+            v_image = SVGMobject(file_name="manim-logo-sidebar.svg")
+            self.add(v_image)
+
+            # An SVG file with multiple objects can be decomposed into
+            # their subcomponents
+            triangle = v_image.submobjects[0]
+            square = v_image.submobjects[1]
+            circle = v_image.submobjects[2]
+            m = v_image.submobjects[3]
+
+            self.play(
+                ApplyMethod(triangle.set_fill, MANIMCE_LOGO_RED),
+                ApplyMethod(square.set_fill, MANIMCE_LOGO_BLUE),
+                ApplyMethod(circle.set_fill, MANIMCE_LOGO_GREEN),
+                ApplyMethod(m.set_fill, WHITE, opacity=1),
+            )
+            self.wait()
+
+Download the resource for the following example `here <https://github.com/ManimCommunity/manim/blob/master/docs/source/_static/favicon.ico>`_ 
+
+.. manim:: ManimCELogoFromImage
+    :save_last_frame:
+
+    class ManimCELogoFromImage(Scene):
+        def construct(self):
+            image = ImageMobject(filename_or_array="favicon.ico")
+            self.add(image)
 
```

Comment:
Maybe here also this logo form this code:
```py
        logo_green = "#87c2a5"
        logo_blue = "#525893"
        logo_red = "#e07a5f"
        ds_m = MathTex(r"\mathbb{M}", z_index=20).scale(7)
        ds_m.shift(2.25*LEFT + 1.5*UP)
        circle = Circle(color=logo_green,
                        fill_opacity=1,
                        z_index=7)
        square = Square(color=logo_blue,
                        fill_opacity=1,
                        z_index=5)
        triangle = Triangle(color=logo_red,
                            fill_opacity=1,
                            z_index=3)
```

Reply:
Do you mean this as another example, or as a way to generate a better image to import in the example? The two examples I've added here are mainly to demonstrate how to use ImageMobject and SVGMobject.

Reply:
As another example. I think it qould be really nice to have the script that creates our logo in the docs